### ISSUE TITLE
Wrong Test Results

### ISSUE THREAD

**Issue by GuoLanqing** [hanyoseob/pytorch-noise2void#2]
I trained the model according to your methods mentioned in readme. Then, I want to test the result of denoising. But the generated denoised image is almost gray. Is there something wrong? And why the sample ratio in test process is 1?

**Comment by ZhidongYang** [comment]
I had the same question. All of the test results are almost gray images. Is there something wrong?

**Comment by Summer-1** [comment]
I have the same question that got gray images. What should I do to solve this question?

**Comment by bensterl15** [comment]
I had the same problem (with grayscale images). Setting the number of channels to 1, normalizing my raw data, and commenting out the colormaps in train.py (starting at line 415) solved it for me.

**Comment by weihua9217** [comment]
I had the same problem.  In my case, I didn't load the pretrained model correctly. 
Replacing "--dir_checkpoint checkpoint", by "--dir_checkpoint checkpoints" may solve it.

**Comment by xiaowowwwww** [comment]
Hello, I would like to know the reason why after modifying the parameters set in the readme, I have been unable to test my image. Thank you very much!
